React/react-workshop/src/components/articleList.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import Article from './article';
+import { fetchArticles } from '../actions/articles.action';
+import { connect } from 'react-redux';
+
+
+class ArticleList extends Component {
+    constructor(props) {
+        super(props);
+        
+        this.state = {
+            posts: []
+        };
+    }
+
+    componentDidMount() {
+        this.props.getArtciles().then((response) => {
+             this.setState({posts: response.articles});
+        })     
+    }
+
+    render() {
+        return (
+            
+            <div className="news-list"> 
+                {this.state.posts.map((post, i) =>
+                        <Article key={post._id} post={post} />
+                )} 
+            </div>
+        );
+    }
+}
+
+export default connect(
+    (state) => {
+        return {
+            articles: state.articles
```

Comment:
Here you are putting articles in props of component but never use it.  The same problem I could see in other components.